src/composables/useContextMenuTranslation.ts

```
@@ -131,11 +131,11 @@ export const useContextMenuTranslation = () => {
       if (matchInput) {
         let match = matchInput[1]
         extraInfo?.inputs?.find((i: INodeInputSlot) => {
```

Biome (2.3.13)

[error] This callback passed to find() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)